test/code-fixtures/projects/ts/src/simpleYetWrong.ts

```diff
@@ -1,5 +1,6 @@
 const ONE = 1;
 const THREE = 3;
+const UNUSED = 4;
```

Comment:
Added this to get another category for filtering by category. Interesting line 11 doesn't show up as unused when scanning.